CHANGELOG.md

```diff
@@ -1,10 +1,40 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-10-15
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
```

Comment:
Feature release:

Image Integrity (https://learn.microsoft.com/en-us/azure/aks/image-integrity?tabs=azure-cli) is now Public Preview